.bumpversion.cfg

```diff
@@ -0,0 +1,12 @@
+[bumpversion]
+current_version = 0.7.3
+commit = True
+tag = True
+
+[bumpversion:file:build.sh]
+search = DOCKER_VERSION=='{current_version}'
+replace = DOCKER_VERSION=='{new_version}'
+
```

Comment:
Bumpversion defines it. Those lines tell bumpversion how to find the version number in the `build.sh` file. It will automatically populate `{current_version}` and `{new_version}`.